Sources/PackageDescription/PackageDescription.swift

```diff
@@ -271,7 +271,7 @@ public final class Package {
         products: [Product] = [],
         dependencies: [Dependency] = [],
         targets: [Target] = [],
-        swiftLanguageVersions: [SwiftVersion]? = nil,
```

Comment:
I think in thise scheme new overload with `swiftLanguageModes` needs default removed as well so we have to versions that accept explicit swiftLanguage* parameter and one that sets a default.